File: Frontend/src/interfaces/v12/StockRecieve/StockRecieveCreateUpdate/StockRecieveCreateUpdateDto.ts
import { guid } from "types/guid";
import { StockRecieveItemCreateUpdateDto } from "./StockRecieveItemCreateUpdateDto";

export interface StockRecieveCreateUpdateDto {
  sellerInfo: string;
  purchaser: string;
  dateOfPurchase: string;
  invoiceNumber?: string;
  warehouseId: guid;
  StockRecieveItems?: StockRecieveItemCreateUpdateDto[];
}

File: Wms.Api/Dto/StockRecieve/StockRecieveCreateUpdate/StockRecieveCreateUpdateDto.cs
namespace Wms.Api.Dto.StockRecieve.StockRecieveCreateUpdate
{
    public class StockRecieveCreateUpdateDto
    {
        public required string SellerInfo { get; set; }
        public required string Purchaser { get; set; }
        public DateTime DateOfPurchase { get; set; }
        public string? InvoiceNumber { get; set; }
        public Guid WarehouseId { get; set; }
        public ICollection<StockRecieveItemCreateUpdateDto>? StockRecieveItems { get; set; }
    }
}
